<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FProduit1</class>
 <widget class="QDialog" name="FProduit1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrix">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelImage">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNom">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSpecialite">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelQuantite">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
